Makefile.dev

```diff
@@ -17,7 +17,7 @@
 
 # Testing the parser -- see parsing/HACKING.adoc
 
-SOURCE_FILES=$(shell git ls-files '*.ml' '*.mli' | grep -v boot/menhir/parser)
+SOURCE_FILES=$(shell git -C $(top_srcdir) ls-files '*.ml' '*.mli' | grep -v boot/menhir/parser)
```

Comment:
Is this a leftover from the previous iteration?

Reply:
Fixed, thanks.